keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -1201,6 +1273,8 @@ This through this both in small and large cases, again with respect to the
 [supported limits]: https://git.k8s.io/community//sig-scalability/configs-and-limits/thresholds.md
 -->
 
+CPU/Memory/Disk will be used when pulling an image volume, proportional to the size of the image.
```

Comment:
An image could be pulled using a constant sized memory buffer.

Reply:
Added that note.